deployments/engine/docker/dind.Dockerfile

```diff
@@ -0,0 +1,13 @@
+# In this file, we build an image with `docker-cli in docker` for engine integration tests.
```

Comment:
Which line indicates we are using `docker:dind`?

Reply:
<img width="675" alt="image" src="https://user-images.githubusercontent.com/61726649/187581388-5dbe2c52-a7f6-4493-86bb-1f4d87a5e86a.png">

Reply:
In jenkins pipeline, we rely on two images: `docker-cli in docker` is built in this file, and `dockerd in docker` is pulled from https://hub.docker.com/_/docker/tags?page=1&name=dind.